futurenhs.app/components/GroupPageHeader/index.tsx

```diff
@@ -21,6 +21,7 @@ import { useMediaQuery } from '@hooks/useMediaQuery'
 import { Theme } from '@appTypes/theme'
 
 import { Props } from './interfaces'
+import { status } from '@constants/status'
```

Comment:
Can this name be made more specific to its purpose?

Reply:
renamed to groupMemberStatus